assets/js/src/listing/notices.jsx

```diff
@@ -0,0 +1,111 @@
+import MailPoet from 'mailpoet';
+import PropTypes from 'prop-types';
+import React, { Fragment } from 'react';
+import ReactStringReplace from 'react-string-replace';
+
+const resumeMailerSending = () => {
+  MailPoet.Ajax.post({
+    api_version: window.mailpoet_api_version,
+    endpoint: 'mailer',
+    action: 'resumeSending',
+  }).done(() => {
+    MailPoet.Notice.success(MailPoet.I18n.t('mailerSendingResumedNotice'));
+    window.mailpoet_listing.forceUpdate();
+  }).fail((response) => {
+    if (response.errors.length > 0) {
+      MailPoet.Notice.error(
+        response.errors.map(error => error.message),
+        { scroll: true }
+      );
+    }
+  });
+};
+
+function MailerError(props) {
+  if (props.mta_log.error && props.mta_log.status === 'paused' && props.mta_log.error.operation !== 'authorization') {
+    if (props.mta_log.error.operation === 'migration') {
+      return (
+        <div className="mailpoet_notice notice notice-warning">
+          <p>{ props.mta_log.error.error_message }</p>
+        </div>
+      );
+    }
+
+    let message = props.mta_log.error.error_message;
+    const code = props.mta_log.error.error_code;
+    if (code) {
+      message += message ? ', ' : '';
+      message += MailPoet.I18n.t('mailerErrorCode').replace('%$1s', code);
+    }
```

Comment:
I may look like some logic is just moved elsewhere but it's not. Please, try to split next time code moving and refactoring in different commits. I's extremely hard to see changes when they are hidden in a single commit that just moves huge blocks. You changed some naming and variables/constants here. Don't hide it. Show it explicitly in a separate commit.

Reply:
Ok